SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SchemaGenerator.cpp

```diff
@@ -207,25 +207,19 @@ void GenerateSubobjectSchemaForActorIncludes(FCodeWriter& Writer, TSharedPtr<FUn
 {
 	TSet<UStruct*> AlreadyImported;
 
-	for (auto& PropertyPair : TypeInfo->Properties)
+	for (const auto& OffsetToSubobject : GetAllSubobjects(TypeInfo))
```

Comment:
Nice! Did you do any validation that the subobjects approach is always a superset of the properties approach?

Reply:
I didn't think this requires additional validation this since `GetAllSubobjects` iterates over `Properties` array and adds them to a map.

Reply:
Oh I didn't see you modified the GetAllSubobjects function also.